github:feature added issue/pr templates (#48)
ShilaTu/esp32-core: .github/ISSUE_TEMPLATE/feature.md

---
name: feature
about: Suggest an idea for this project
title: 'feature: '
labels: enhancement
assignees: ''

---

> **Is your feature request related to a problem? Please describe.**

Author: ```suggestion
**Description**
```

A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**

> A clear and concise description of what you want to happen.

Author: ```suggestion
**Solution**
A clear and concise description of your preferred solution.
```

**Describe alternatives you've considered**

> A clear and concise description of any alternative solutions or features you've considered.

Author: ```suggestion
**Alternative solutions**
A clear and concise description of any alternative solutions or features you've considered.
- first alternative solution
- second alternative solution
```

**Additional context**

> Add any other context or screenshots about the feature request here.

Author: ```suggestion
**Context**
If applicable, add screenshots or code to help explain your problem and any other context about the problem here.
```
